src/app/features/container/utils/route-helpers.ts

```diff
@@ -12,7 +12,7 @@ function isHomePage(pathname: RouteUrls) {
 }
 
 export function isLandingPage(pathname: RouteUrls) {
-  return pathname === RouteUrls.RequestDiagnostics || pathname.match(RouteUrls.Onboarding); // need to match get-started/ledger
+  return pathname === pathname.match(RouteUrls.Onboarding); // need to match get-started/ledger
```

Comment:
```suggestion
  return pathname.match(RouteUrls.Onboarding); // need to match get-started/ledger
```

I can clean this up further as I will refactor this area